byway/src/main/webapp/home.js

```diff
@@ -0,0 +1,8 @@
+window.onload = function(){
```

Comment:
I think the `DOMContentLoaded` event (or whatever it is) that we've discussed previously is better than window.onload.

Reply:
changed to this